.github/workflows/comment-triggered-ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
   scaled_sim:
     uses: ./.github/workflows/run-on-comment.yml
     with:
-      runs-on: [self-hosted, test]
+      runs-on: "[self-hosted, test]"
```

Comment:
Out of curiosity, why is this a string of a list?  According to [the documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idruns-on) the value should be a single string, or a variable containing a list, or an array of strings/variable containing strings,or key-value pairs.